### DISCUSSION TITLE
Cuda and Multicore Backends

### DISCUSSION THREAD

**Discussion by ted-kord** [google-research/dex-lang#656] [Q&A] [answered]
I have raised an [issue](https://github.com/google-research/dex-lang/issues/651) regarding this but I would be grateful if someone could have a look or indicate what I am doing wrong.

I am still unable to use either the "llvm-mc" or "llvm-cuda" options. The multicore backend crashes with a `segmentation fault (core dumped)` while the 'cuda'  option is not part of the selectable backends : `option --backend: Bad option. Expected one of: ["llvm","llvm-mc","mlir","interpreter"]`.

**Comment by apaszke** [comment, marked as answer]
Sorry, we're aware of this bug, but we're tied up with some other work (not all Dex related). We can't guarantee quick turnaround for bug fixes at this point. We'll try to get to it, but I can't give you an estimate just now.

**Reply by ted-kord** [reply]
Thank you for the update. I will be patient.
